packages/halo-theme/src/custom-elements/ef-calendar.less

```diff
@@ -83,6 +83,10 @@
     &:not(:active) [part~=selection]:hover {
       border-color: @calendar-cell-hover-background-color;
     }
+    &[today][selected] [part~=selection]:hover,
```

Comment:
Why? Is it not covered at line 106?

Reply:
Line 86 need to selector only today with selected
But line 106 is selector for some of [selected, today, range]

Reply:
OK, understood